scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableConfig.scala

```diff
@@ -93,7 +93,7 @@ object UnlessInsideBlock {
   implicit val reader: ConfDecoder[UnlessInsideBlock] =
     ConfDecoder.instanceF[UnlessInsideBlock] {
       case c: Conf.Obj =>
-        (c.get[DisabledSymbol]("safeBlock") |@|
+        (c.get[List[DisabledSymbol]]("safeBlock") |@|
```

Comment:
This change breaks existing configuration where a single safeBlock is defined `safeBlock = "a.B"`. We should allow users to either declare a single string or an array. We can do this by writing a custom decoder, something like
```
  val safeBlockReader = ConfDecoder.instanceF[List[DisabledSymbol]] { conf =>
    val toRead = conf match {
      case str @ Conf.Str(_) => Conf.Lst(str)
      case e => e
    }
    ConfDecoder[List[DisabledSymbol]].read(toRead)
  }
  implicit val reader: ConfDecoder[UnlessInsideBlock] =
    ConfDecoder.instanceF[UnlessInsideBlock] {
      case c: Conf.Obj =>
        (
          c.get[DisabledSymbol]("safeBlock") |@|
            c.get[List[DisabledSymbol]]("symbols")(safeBlockReader) // <- pass in custom decoder
```

Reply:
I'm fine with the config breaking, but sure it would be useful if it supported both formats.

Reply:
I agree it's not critical to keep compat in this particular case but I would like to keep stability when possible, and having both formats would be nice indeed.

Reply:
perhaps the field should be renamed to `safeBlocks` though... then I think it's clear that it expects a list.

Reply:
Hi @olafurpg and @fommil, thanks for your comments! To recap, we agree on:

- ensuring compatibility
- renaming to `safeBlocks`

Is there something else we should take into account?